src/eventsAfter.ts

```diff
@@ -68,17 +68,17 @@ async function getRemoteMessagesSince (src: string, contractID: string, sinceHei
   const response = await fetch(`${src}/eventsAfter/${contractID}/${sinceHeight}`)
   if (!response.ok) {
     // The response body may contain some useful error info if we got a Boom error response.
-    const bodyText = await response.text().catch(_ => '') || ''
+    const bodyText = await response.text().catch(_ => '') !== undefined ? await response.text().catch(_ => '') : ''
```

Comment:
This is really weird, can you revert this is as well?

Reply:
Not only is this weird, but it's also wrong, as you can only call `.text()` once (because it consumes the response body).